src/Stratis.Bitcoin.Features.Miner.Tests/StakeSplittingTests.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using FluentAssertions;
+
+using NBitcoin;
+
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.Miner.Staking;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Tests.Common;
+using Xunit;
+using StakingWallet = Stratis.Bitcoin.Features.Wallet.Wallet;
+
+
+namespace Stratis.Bitcoin.Features.Miner.Tests
+{
+    public class StakeSplittingTests : PosMintingTest
+    {
+        private const int ChainHeight = 500000;
+
+        private const int SplitFactor = 8;
+
+        public StakeSplittingTests()
+        {
+            this.network.Consensus.Options = new PosConsensusOptions();
+        }
+
+        [Fact]
+        public void Given_A_Wallet_With_Big_And_Small_Coins_Then_Big_Coins_Below_Target_Should_Get_Split()
+        {
+            //target value around 4700
```

Comment:
We prefer `// Text.` over `//text`, therefore this should be `// Target value around 4700.` or `// Aiming for target value around 4700.`